### ISSUE TITLE
New line in media queries

### ISSUE THREAD

**Issue by szepeviktor** [jakubpawlowicz/clean-css#834]
Could we leave the first new line in with `-b`?

removed
```css
@media (min-width:576px){.navbar{border-radius:.25rem}
.navbar-fixed-bottom,.navbar-fixed-top,.navbar-full,.navbar-sticky-top{border-radius:0}
}
```

proposed
```css
@media (min-width:576px){
.navbar{border-radius:.25rem}
.navbar-fixed-bottom,.navbar-fixed-top,.navbar-full,.navbar-sticky-top{border-radius:0}
}
```

**Comment by jakubpawlowicz** [comment]
Sure, that would make more sense!

**Comment by jakubpawlowicz** [comment]
Fixed on master, due in 4.0.

**Comment by szepeviktor** [comment]
Thank you.
